### DISCUSSION TITLE
Add rule only if variable is in a class

### DISCUSSION THREAD

**Discussion by arome** [Igorkowalski94/eslint-plugin-project-structure#22] [Help] [answered]
Hello, I'm looking to allow variable to have this format "_{camelCase}", only if they're nested in a class. How can I write a rule that will only allow it in class but not in arrowFunctions?

**Comment by Igorkowalski94** [comment]
Hi @arome, 

Currently, this option is not available unless your class files have a prefix in the file name or your classes are located in a folder with a specific name. 

Which specific case are you interested in? If it’s case1, I can implement it relatively easily in the next version by introducing a new `propertyDefinition` selector. If it’s case2, it will require a bit more work.

```ts
class Car {
  _model: string; // case1

  constructor(model: string) {
    this._model = model;
  }

  doSomething = (): string => {
    const _someVariable = ""; // case2

    return `${_someVariable} ${this._model}`;
  }
}
```

**Reply by Igorkowalski94** [reply, marked as answer]
@arome 

[v3.9.0](https://github.com/Igorkowalski94/eslint-plugin-project-structure/releases/tag/v3.9.0) Now you can define case1 using the new selector `propertyDefinition`.

**Reply by arome** [reply]
Hey thanks for your quick response and update!
It is indeed case1 that I was looking for
